llm-models/embedding/automl/automl_embeddings_example.py

```diff
@@ -0,0 +1,147 @@
+# Databricks notebook source
+# MAGIC %md # AutoML Embeddings with LangChain
+# MAGIC
+# MAGIC LangChain is a tool that allows you to easily connect large language models with vector stores. In this specific example, we are assuming we have trained an embeddings model with AutoML Embeddings and have the `run_id` of the experiment on hand.
+# MAGIC
+# MAGIC Requirements
+# MAGIC - Databricks Runtime ML 13.1 and above
+# MAGIC - (Recommended) GPU instances
+
+# COMMAND ----------
+
+# MAGIC %pip install -q faiss-cpu
```

Comment:
I bit counterintuitive that the requirements recommend GPU instances but here faiss-cpu is installed. Is this intentional? I recommend clarifying what's going on here.

Reply:
I'll swap it out with faiss-gpu